src/test/scala/mesosphere/marathon/api/v2/json/AppUpdateFormatTest.scala

```diff
@@ -45,13 +37,6 @@ class AppUpdateFormatTest extends UnitTest {
       appUpdate.acceptedResourceRoles should equal(Some(Set(ResourceRole.Unreserved)))
     }
 
-    "FromJSON should fail when 'acceptedResourceRoles' is defined but empty" in {
-      val json = """ { "id": "test", "acceptedResourceRoles": [] }"""
```

Comment:
Is this still true for our update? Does the validator fail when we merge this update with an app?

Reply:
Yes, check this line:

https://github.com/mesosphere/marathon/blob/master/src/test/scala/mesosphere/marathon/api/v2/json/AppDefinitionFormatsTest.scala#L213